diff -urN kile-1.8.org/src/kile/kile.desktop kile-1.8/src/kile/kile.desktop
--- kile-1.8.org/src/kile/kile.desktop	2005-05-21 13:17:52.000000000 +0200
+++ kile-1.8/src/kile/kile.desktop	2005-05-22 17:57:35.255232720 +0200
@@ -19,6 +19,7 @@
 Comment[hu]=(La)TeX fejlesztőkörnyezet
 Comment[it]=Ambiente di sviluppo (La)TeX
 Comment[nl]=(La)TeX-ontwikkelingsomgeving
+Comment[pl]=Środowisko do tworzenia w (La)TeXu
 Comment[pt]=Ambiente de desenvolvimento de (La)TeX
 Comment[pt_BR]=Ambiente de Desenvolvimento (La)TeX
 Comment[sr]=Развојно окружење за (La)TeX
@@ -54,5 +55,5 @@
 GenericName[tr]=LaTeX Arayüzü
 GenericName[zh_CN]=LaTeX 前端
 MimeType=text/x-tex;text/x-kilepr
-Categories=Qt;KDE;Office;
+Categories=Qt;KDE;TextEditor;
 Encoding=UTF-8
