--- kile-2.0.1/src/kile/kile.desktop.orig	2007-11-23 14:07:01.000000000 +0100
+++ kile-2.0.1/src/kile/kile.desktop	2008-06-20 16:50:22.000000000 +0200
@@ -27,6 +27,7 @@
 Comment[nds]=(La)TeX-Dokmentensmeed
 Comment[nl]=(La)TeX-ontwikkelingsomgeving
 Comment[pa]=(La)TeX ਵਿਕਾਸ ਵਾਤਾਵਰਣ
+Comment[pl]=Środowisko do tworzenia w (La)TeXu
 Comment[pt]=Ambiente de desenvolvimento de (La)TeX
 Comment[pt_BR]=Ambiente de Desenvolvimento (La)TeX
 Comment[sk]=(La)TeX vývojové prostredie
@@ -65,6 +66,7 @@
 GenericName[nds]=LaTeX-Böversiet
 GenericName[nl]=LaTeX-tekstverwerking
 GenericName[pa]=LaTeX ਮੁੱਖ
+GenericName[pl]=Interfejs dla LaTeXa
 GenericName[pt]=Interface para o LaTeX
 GenericName[pt_BR]=Interface do LaTeX
 GenericName[ru]=Интерфейс к LaTeX
@@ -78,5 +80,5 @@
 GenericName[xx]=xxLaTeX Frontendxx
 GenericName[zh_CN]=LaTeX 前端
 MimeType=text/x-tex;text/x-kilepr;
-Categories=Qt;KDE;Office;
+Categories=Qt;KDE;TextEditor;
 Encoding=UTF-8
